src/GraphQL/Transports/IGraphQLTextSerializer.cs

```diff
@@ -0,0 +1,26 @@
+using GraphQL.Transports.Json;
+
+namespace GraphQL
+{
+    /// <summary>
+    /// Serializes and deserializes object hierarchies to/from a string.
+    /// Should include special support for <see cref="ExecutionResult"/>, <see cref="Inputs"/>
+    /// and transport-specific classes as necessary.
+    /// Typical JSON-specific classes are providied within <see cref="Transports.Json">GraphQL.Transports.Json</see>.
+    /// </summary>
+    public interface IGraphQLTextSerializer : IGraphQLSerializer
```

Comment:
Why did this interface need? It seems redundant and reminds the case when the task can be solved by the extension methods that calls methods from the main (base) interface.

Reply:
String-based serializing and deserializing is handled natively by both `Newtonsoft.Json` and `System.Text.Json` and always executes synchronously.  Nor is there any reason that a binary serializer should ever convert to a string.  I believe it improper for part of the main project to have extension methods to provide such a conversion.

See removed method `WriteToStringAsync`.

Reply:
OK. So we expose these methods from interface because:
1. We can 😄 
2. For `public static async Task<string> ExecuteAsync(this ISchema schema...` extension method which in turn is used by other two extension methods from serialization projects - ` public static Task<string> ExecuteAsync(this ISchema schema...` That is, these auxiliary methods that we do not use anywhere.